src/jit/emit.h

```diff
@@ -2159,10 +2161,12 @@ inline bool emitter::emitIsScnsInsDsc(instrDesc* id)
 
 inline insUpdateModes emitter::emitInsUpdateMode(instruction ins)
 {
-#ifdef DEBUG
+#ifdef _TARGET_XARCH_
+    return InstructionInfo::lookupUpdateMode(ins);
```

Comment:
I'm not a fan of the verb "lookup". IMO, getting an entry in a table is the perfect place for the index (`[]`) operator - in fact, I'm often inclined to overload that operator even when it's not actually an array.
If a specific "get" method is desired, I would use that verb, e.g. `getUpdateMode(ins)`.

Reply:
You can't actually have static index operators in C++, I've changed the name to just `get()` for the time being.